### DISCUSSION TITLE
LGPL Exclusion licenses

### DISCUSSION THREAD

**Discussion by ghost** [itas109/CSerialPort#93] [Ideas]
Hi,
I would like to know if you are providing LGPL exclusion license. and under what terms?

**Comment by itas109** [comment]
Only LGPL now. Is there any need？

**Comment by ghost** [comment]
I integrated to into app, this app does real-time serial-port I/O with certain legacy medical devices.

Due to the amount of data passed, the app requires higher performance, and thus, linking static instead of putting your code inside a DLL.

**Comment by ghost** [comment]
Any update?

**Reply by itas109** [reply]
I'm going to change the open source license to dual licensing(LGPL and commercial licenses). I need some time.

**Comment by itas109** [comment]
CSerialPort is licensed under the LGPLv3 with LGPL-3.0-linking-exception.
